airflow/serialization/serde.py

```diff
@@ -337,6 +344,11 @@ def _is_pydantic(cls: Any) -> bool:
     return hasattr(cls, "model_config") and hasattr(cls, "model_fields") and hasattr(cls, "model_fields_set")
 
 
+def _is_namedtuple(cls: Any) -> bool:
+    """Return True if the class is a namedtuple."""
+    return isinstance(cls, tuple) and hasattr(cls, "_asdict") and hasattr(cls, "_fields")
```

Comment:
isinstance is slow. Is it possible to just check on attributes?

Reply:
Yes, done